skrf/calibration/deembedding.py

```diff
@@ -2572,7 +2705,11 @@ def split2xthru(self, se_2xthru, se_fdf):
                                   NRP_enable = self.NRP_enable,
                                   leadin = self.leadin,
                                   verbose = self.verbose)
-
+        # debug outpus
+        self.gamma_cc = dm_cc.gamma
+        self.x_end_cc = dm_cc.x_end
+        self.z_side1_cc = dm_cc.z_side1
+        self.z_side2_cc = dm_cc.z_side2

```

Comment:
still needed? (same above)

Reply:
I think it is interesting for users who either want to check what happened internally or want to do a port-extension of their fixture. Should the need arise like in #1075  they have the gamma and the index to read the last impedance of the fixture. They can build from this a transmission line for common and differential modes and convert back to single-ended. We could include a dedicated method for that if there is more demand, but it is outstide the scope of this PR.